gravatar-quickeditor/build.gradle.kts

```
@@ -132,16 +133,47 @@ dependencies {
     testImplementation(project(":uitestutils"))
 }
 
-project.afterEvaluate {
-    publishing {
-        publications {
-            create<MavenPublication>("maven") {
-                from(components["release"])
+val sdkVersion: String by rootProject.extra
 
-                groupId = "com.gravatar"
-                artifactId = "gravatar-quickeditor"
-                // version is set by `publish-to-s3` plugin
+mavenPublishing {
+    publishToMavenCentral(SonatypeHost.CENTRAL_PORTAL)
+
+    coordinates(
+        groupId = "com.gravatar",
+        artifactId = "gravatar-quickeditor",
+        version = sdkVersion,
+    )
+
+    pom {
+        name.set("Gravatar Android SDK")
+        description.set("The official Gravatar Android SDK")
+        url.set("https://github.com/Automattic/Gravatar-SDK-Android")
+        licenses {
+            license {
+                name.set("Mozilla Public License, Version 2.0")
+                url.set("http://www.mozilla.org/MPL/2.0/index.txt")
+            }
+        }
+
+        scm {
+            connection.set("scm:git:github.com:Automattic/Gravatar-SDK-Android.git")
+            developerConnection.set("scm:git:ssh://github.com:Automattic/Gravatar-SDK-Android.git")
```

WDYT about not suggest using ssh - I think clone by https (especially of open source project) is rather default
```suggestion
            developerConnection.set("scm:git:https://github.com/Automattic/Gravatar-SDK-Android.git")
```